_posts/2024-04-02-Multilingual-Search.md

```diff
@@ -0,0 +1,320 @@
+---
+layout: post
+title:  "Multilingual search with OpenSearch"
+authors:
+- roigam
+date: 2024-04-01 13:30:00 -0600
```

Comment:
Please update the publish date to 4/25/2024

Reply:
done